library/src/scripts/components/forms/select/SearchBar.tsx

```diff
@@ -119,7 +121,7 @@ export default class SearchBar extends React.Component<IProps, IState> {
                 menuIsOpen={this.isMenuVisible}
                 classNamePrefix={this.prefix}
                 className={classNames(this.prefix, className)}
-                placeholder={this.props.placeholder}
+                placeholder={this.props.placeholder || ""}
```

Comment:
It looks like you're using `||` to implement a default value for this property. Would it be better to set `placeholder` to an empty string in `defaultProps`?